lisp_ease/src/lisp-ease.asd

```diff
@@ -0,0 +1,32 @@
+(asdf:defsystem lisp-ease
+	:depends-on (roslisp
+               cram-language
+               cram-json-prolog
+               std_msgs-msg
+               cl-transforms
+               cl-transforms-stamped
+               cl-tf
+               cram-bullet-world-tutorial)
+	:components
+	
+	((:module "queries"
```

Comment:
Very important!
The ordering of files should be according to their dependencies. Also, **always** specify file interdependencies. Also, consider one line comments to describe which file does what very briefly.

Reply:
Done